benchmark/schema/operator-distributed.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "operator-distributed.json",
+  "title": "Distributed Operator",
+  "description": "Reusable distributed operator definition",
+  "type": "object",
+  "properties": {
+    "filename": {
+      "type": "string"
+    },
+    "stencil": {
```

Comment:
this will also lead the format changes. Is it intentional?
previously,
```
"stencil": "5pt",
"local_size" ....
```
now
```
"stencil": {
  "name": "5pt",
  ...
}
```

Reply:
Yes, because I find this more consistent. But if others prefer the old version, I can revert.